tests/integration/conftest.py

```diff
@@ -204,3 +204,14 @@ def use_service_colors():
 
     # Code here runs after the test, reverting the state
     pyansys_geometry.USE_SERVICE_COLORS = False
+
+
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(scope="function")
```
Let's make this fixture explicitly state that it is run on each test it is called